docs/integrations-in-rancher/cloud-marketplace/aws-cloud-marketplace/install-adapter.md

```diff
@@ -110,18 +112,16 @@ For the below instructions, replace `$MY_ACC_NUM` with your AWS account number a
   <TabItem value="Private CA Authority / Rancher-generated Certificates">
 
   ```bash
-  helm install rancher-csp-adapter rancher-charts/rancher-csp-adapter --namespace cattle-csp-adapter-system --set aws.enabled=true --set aws.roleName=$MY_ROLE_NAME --set-string aws.accountNumber=$MY_ACC_NUM --set additionalTrustedCAs=true --set image.tag=v1.0.1
+  helm install rancher-csp-adapter rancher-charts/rancher-csp-adapter --namespace cattle-csp-adapter-system --set aws.enabled=true --set aws.roleName=$MY_ROLE_NAME --set-string aws.accountNumber=$MY_ACC_NUM --set additionalTrustedCAs=true
   ```
 
   Alternatively, you can use a `values.yaml` and specify options the below:
 
   ```yaml
-  image:
-  tag: v1.0.1
```

Comment:
Are there separate 2.6 and 2.7 docs for csp adapter ? If so it seems like we should leave this in the 2.6 version and change it in the 2.7 version.

Reply:
This would only be specified in the 2.6.7/2.6.8 install, we wouldn't want people to do this in 2.6.9 and above. There also does not appear to be split docs for 2.6/2.7 for the adapter, although I can try and do that if that's something that we want.

I'm considering the following options for the special 2.6.7/2.6.8 instructions:
- Have a * in the table that points them to a special install instructions at the bottom of the doc, have only normal instructions in the doc 
- Remove the mention of 2.6.7/2.6.8, tell people to only use 2.6.9 and higher with the csp adapter 

So far, I've just left an "Important" block that tells people what they need to do for 2.6.7/2.6.8, let me know if you think that won't suffice.

Reply:
I've updated the release/v2.7 branch so that there's now a separate folder for 2.6.

/docs for to 2.6
/versioned_docs/version-2.6 for 2.6

Reply:
Thanks @btat, I'll re-work this PR to use those versioned docs instead. 

There were a couple of things that I noticed about the versioned docs (can open a new issue for this if you would prefer):
- There's a message on the 2.6 docs that says: "This is documentation for Rancher Manager v2.6, which is no longer actively maintained". This is not true - v2.6 has not reached EOL or EOM - it is actively maintained. I think this message was just copied over from the 2.5 docs, I would recommend updating it to be more accurate.
- On startup, I see a bunch of warnings related to broken links. I've supplied them below. It might be worth looking into this to see if there's something wrong.

```
[WARNING] Docs markdown link couldn't be resolved: (../../../../versioned_docs/version-2.0-2.4/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/upgrades/namespace-migration.md) in /home/root/versioned_docs/version-2.6/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/air-gapped-upgrades.md for version 2.6
rancher-docs-docusaurus-1  | [WARNING] Docs markdown link couldn't be resolved: (../../../../versioned_docs/version-2.0-2.4/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/restore-rancher-launched-kubernetes-clusters-from-backup.md) in /home/root/versioned_docs/version-2.6/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/rollbacks.md for version 2.6
rancher-docs-docusaurus-1  | [WARNING] Docs markdown link couldn't be resolved: (../../../../versioned_docs/version-2.0-2.4/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/restore-rancher-launched-kubernetes-clusters-from-backup/roll-back-to-v2.0-v2.1.md) in /home/root/versioned_docs/version-2.6/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/rollbacks.md for version 2.6
rancher-docs-docusaurus-1  | [WARNING] Docs markdown link couldn't be resolved: (../../../../versioned_docs/version-2.0-2.4/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/upgrades/helm2.md) in /home/root/versioned_docs/version-2.6/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/upgrades.md for version 2.6
rancher-docs-docusaurus-1  | [WARNING] Docs markdown link couldn't be resolved: (../../../../versioned_docs/version-2.0-2.4/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/upgrades/namespace-migration.md) in /home/root/versioned_docs/version-2.6/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/upgrades.md for version 2.6
rancher-docs-docusaurus-1  | [WARNING] Docs markdown link couldn't be resolved: (../../../../../versioned_docs/version-2.0-2.4/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/upgrades/namespace-migration.md) in /home/root/versioned_docs/version-2.6/getting-started/installation-and-upgrade/other-installation-methods/rancher-on-a-single-node-with-docker/upgrade-docker-installed-rancher.md for version 2.6
rancher-docs-docusaurus-1  | [WARNING] Docs markdown link couldn't be resolved: (../../../../versioned_docs/version-2.0-2.4/pages-for-subheaders/helm2.md) in /home/root/versioned_docs/version-2.6/getting-started/installation-and-upgrade/resources/helm-version-requirements.md for version 2.6
rancher-docs-docusaurus-1  | [WARNING] Docs markdown link couldn't be resolved: (../../versioned_docs/version-2.0-2.4/pages-for-subheaders/cluster-logging.md) in /home/root/versioned_docs/version-2.6/pages-for-subheaders/gke-cluster-configuration.md for version 2.6
rancher-docs-docusaurus-1  | [WARNING] Docs markdown link couldn't be resolved: (../../../versioned_docs/version-2.0-2.4/explanations/integrations-in-rancher/notifiers.md) in /home/root/versioned_docs/version-2.6/reference-guides/pipelines/pipeline-configuration.md for version 2.6
****
```